graphs/adapter-script-generation/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/scriptgen/GraphScriptGenCheckFactory.kt

```diff
@@ -47,7 +52,7 @@ object GraphScriptGenCheckFactory : GraphCheckFactory {
     ) :
         GetVertexCountBehaviour(), ScriptGenerationCheck {
         override fun getCheckScript() =
-            GraphScriptingHelper.scriptifyFunction(GraphChecks::hasVertexCount, graphName, from, to, shortestPath)
+            GraphScriptingHelper.scriptifyFunction(GraphChecks::hasShortestPath, graphName, from, to, shortestPath)
```

Comment:
⚠️ You're changing this in another PR as well, so you might and up with a conflict between the two...